stable/dex/values.yaml

```diff
@@ -155,9 +171,9 @@ config:
 #    secret: ZXhhbXBsZS1hcHAtc2VjcmV0
 #
   enablePasswordDB: true
-# staticPasswords:
-#  - email: "admin@example.com"
-#    # bcrypt hash of the string "password"
-#    hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W"
-#    username: "admin"
-#    userID: "08a8684b-db88-4b73-90a9-3cd1661f5466"
+#  staticPasswords:
+#   - email: "admin@example.com"
+#     # bcrypt hash of the string "password"
+#     hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W"
+#     username: "admin"
+#     userID: "08a8684b-db88-4b73-90a9-3cd1661f5466"
```

Comment:
This is now in a configmap not a secret.... This seems like a major regression.

Reply:
@kfox1111 as I've mentioned in the PR notes above had to do that to have more control over config contents. Secret requires all the data inside to be base64 encrypted and I wasn't able to find any nice way how to perform multiline base64 enc in Helm.
Please advise if you know such.

Otherwise I will rollback to the Secret but dex config data (specifically port numbers, http/https choice) will have to be populated manually like it was done before. Moreover with the previous version of chart values there was no possibility to disable dex http listener, cause it has been hardcoded.